.github/workflows/test.yml

```diff
@@ -18,18 +18,18 @@ jobs:
     container:
       image: ubuntu:focal
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         path: '.'
     - uses: actions/setup-node@v3.5.1
       with:
         node-version: '12.x'
     - run: .github/workflows/build-and-test.sh
-    - uses: ros-tooling/setup-ros@v0.2
-    - uses: actions/checkout@v2
+    - uses: ros-tooling/setup-ros@v0.6
```

Comment:
`v0.7` is available

```suggestion
    - uses: ros-tooling/setup-ros@v0.7
```

Reply:
with version 0.7 it does not compile.
you must first perform an operation to remove the deprecated functions.